app/assets/stylesheets/fae/modules/tables/_image.scss

```diff
@@ -0,0 +1,19 @@
+th {
+  &.-image {
+    width: 120px;
+  }
+}
+
+td {
+  .image-mat {
```

Comment:
This property doesn't need to be nested. Additionally, in `_actions.scss`, there's a rule for `&.-thumb` on nested table columns that have an image. Move that rule into this file please, for consistency's sake.